ios/Sources/FavoritesFeature/FavoritesFeature.swift

```diff
@@ -51,5 +54,8 @@ public let favoritesReducer = Reducer<FavoritesState, FavoritesAction, Favorites
         return .none
     case .favorite:
         return .none
+    case let .toggleSettings(isPresented):
+        state.isSettingPresented = isPresented
+        return .none
```

Comment:
I think this naming is a little hard to understand what is it used for.

So please rename this action or divide this action to `showSettings` and `hideSettings`

Reply:
OK

Reply:
I know that it's delicate problem but I think it's `showSetting` not `showSettings` also for hide.